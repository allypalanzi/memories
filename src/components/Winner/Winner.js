import React from 'react';
import styles from './Winner.scss';

const Winner = props => <h3 className={styles.winner}>🎉 You win! 🎉</h3>;

export default Winner;
