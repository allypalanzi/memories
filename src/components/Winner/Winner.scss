@keyframes fadein {
  from { opacity: 0; }
  to   { opacity: 1; }
}

.winner {
  position: absolute;
  top: 50%;
  z-index: 2;
  left: 50%;
  font-size: 5em;
  transform: translate(-50%, -50%);
  width: 100%;
   animation: fadein 200ms;
}
