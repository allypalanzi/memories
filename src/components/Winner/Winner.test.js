import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Winner from './Winner.js';

describe('<Winner />', () => {

  const renderedComponent = shallow(<Winner />);

  it('renders correctly', () => {
    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
