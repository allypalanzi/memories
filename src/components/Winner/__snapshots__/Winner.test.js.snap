// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Winner /> renders correctly 1`] = `
<h3
  className="winner"
>
  🎉 You win! 🎉
</h3>
`;
